opm/autodiff/BlackoilWellModel_impl.hpp

```diff
@@ -29,6 +29,10 @@ namespace Opm {
 
         extractLegacyCellPvtRegionIndex_();
         extractLegacyDepth_();
+
+        const Grid& grid = ebosSimulator_.gridManager().grid();
+        const auto eclipse_grid = Opm::UgGridHelpers::createEclipseGrid(grid, ebosSimulator.gridManager().eclState().getInputGrid());
```

Comment:
It looks like this grid is just part of the grid on this partition for parallel running. Any suggestion for here?

Reply:
Does the following code do the trick?
```
100         globalGrid_ = simulator_.gridManager().grid();
101         globalGrid_.switchToGlobalView();
```

Reply:
Yes. this will fix the parallel running.